LiveSplit.AutoSplitters.xml

```diff
@@ -14079,4 +14079,14 @@
         	<Type>Script</Type>
         	<Description>AutoSplitter</Description>
     	</AutoSplitter>
+	<AutoSplitter>
+        <Games>
+            <Game>Dying Light 2: Stay Human</Game>
+        </Games>
+        <URLs>
+            <URL>https://github.com/TheDementedSalad/Dying-Light-2-Load-Remover/blob/main/Dying%20Light%202%20Load%20Remover.asl</URL>
```

Comment:
Please use the raw URL instead.

Reply:
Sorry I think I'm being dumb here, what do you mean the raw URL?

Reply:
Oh shit yeah my bad